README.md

```diff
@@ -57,7 +57,7 @@ GlobalID::Locator.locate_signed person_sgid
 
 **Expiration**
 
-Signed Global IDs can expire some time in the future. This is useful if there's a resource
+Signed Global IDs can expire sometime in the future. This is useful if there's a resource
```

Comment:
```suggestion
Signed Global IDs can expire some time in the future. This is useful if there's a resource
```

Reply:
Hi @rafaelfranca 
_Grammarly_ suggested to fix this. See description
![Screenshot from 2024-05-30 00-29-04](https://github.com/rails/globalid/assets/6830119/c6853d3b-d73c-4878-8239-7769620422f5)
Do you want me to commit your suggestion instead?